lua/nvim-treesitter/ts_utils.lua

```diff
@@ -134,7 +134,19 @@ function M.get_node_at_cursor(winnr)
   if not root_lang_tree then
     return
   end
-  local root = M.get_root_for_position(cursor_range[1], cursor_range[2], root_lang_tree)
+
+  local root
+  if ignore_injected_langs then
+    for _, tree in ipairs(root_lang_tree:trees()) do
```

Comment:
this is the same loop as is done in `get_node_at_position`. i thought about extracting into a method but couldn't think of a name which wasn't like basically like `get_root_for_position` lol